src/app/GitUI/Themes/highcontrast_dark.css

```diff
@@ -4,11 +4,10 @@
 .OtherTag                    { color: #a1a1a1; }
```

Comment:
move, too?

Reply:
yes, I plan to remove that theme so it does not make much difference...